homeassistant/components/zwave_js/cover.py

```diff
@@ -5,7 +5,7 @@
 from typing import Any
 
 from zwave_js_server.client import Client as ZwaveClient
-from zwave_js_server.const import BarrierState
+from zwave_js_server.const.command_class.barrior_operator import BarrierState
```

Comment:
barrior?

Reply:
🤦🏾 thanks for catching that, it'll get resolved in our next dependency bump 🙂